### PR TITLE
[Microsoft.BillingBenefits] Initializing new service

### PR DESCRIPTION
**Pull request by gaoyp830**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by Yao725 — COMMENTED** [review]
Hi @gaoyp830, is this namespace approved by the arch board? If not, you can open an issue (example: https://github.com/Azure/azure-sdk/issues/5026) in [azure-sdk](https://github.com/Azure/azure-sdk) to trigger the namespace review process.

**Comment by gaoyp830** [comment]
> Hi @gaoyp830, is this namespace approved by the arch board? If not, you can open an issue (example: [Azure/azure-sdk#5026](https://github.com/Azure/azure-sdk/issues/5026)) in [azure-sdk](https://github.com/Azure/azure-sdk) to trigger the namespace review process.

Created request: https://github.com/Azure/azure-sdk/issues/5181

**Comment by gaoyp830** [comment]
> Hi @gaoyp830, is this namespace approved by the arch board? If not, you can open an issue (example: [Azure/azure-sdk#5026](https://github.com/Azure/azure-sdk/issues/5026)) in [azure-sdk](https://github.com/Azure/azure-sdk) to trigger the namespace review process.

@Yao725  I have got namespace approval for .net https://github.com/Azure/azure-sdk/issues/5181, please continue reviewing this PR

File: sdk/billingbenefits/Azure.ResourceManager.BillingBenefits/README.md
Comment (Yao725) [inline sdk/billingbenefits/Azure.ResourceManager.BillingBenefits/README.md:3]: Please remove the locale information of the link.

File: sdk/billingbenefits/Azure.ResourceManager.BillingBenefits/tests/Scenario/ReservationOrderAliasTests.cs
Comment (Yao725) [inline sdk/billingbenefits/Azure.ResourceManager.BillingBenefits/tests/Scenario/ReservationOrderAliasTests.cs:18]: I think it's better to use field here.
```suggestion
        private ReservationOrderAliasModelResource _modelResource;
        private TenantResource _tenantResource;
```

File: sdk/billingbenefits/Azure.ResourceManager.BillingBenefits/tests/Scenario/ReservationOrderAliasTests.cs
Comment (Yao725) [inline sdk/billingbenefits/Azure.ResourceManager.BillingBenefits/tests/Scenario/ReservationOrderAliasTests.cs:184]: Maybe it's better to move this method to `TestHelpers`?

File: sdk/billingbenefits/Azure.ResourceManager.BillingBenefits/tests/Scenario/ReservationOrderAliasTests.cs
Comment (Yao725) [inline sdk/billingbenefits/Azure.ResourceManager.BillingBenefits/tests/Scenario/ReservationOrderAliasTests.cs:199]:
```suggestion
            var request = new ReservationOrderAliasModelCreateOrUpdateContent(new BillingBenefitsSku("Standard_B1s"))
            {
                BillingScopeId = "/subscriptions/eef82110-c91b-4395-9420-fcfcbefc5a47",
                Term = new Term("P3Y"),
                AppliedScopeType = scope,
                DisplayName = "TestROName" + scope.ToString(),
                BillingPlan = new BillingPlan("P1M"),
                ReservedResourceType = "VirtualMachines",
                Renew = false,
                Quantity = 1,
                Location = AzureLocation.EastUS,
                ReservedResourceProperties = new ReservationOrderAliasRequestReservedResourceProperties
                {
                    InstanceFlexibility = InstanceFlexibility.On
                }
            };
```

File: sdk/billingbenefits/Azure.ResourceManager.BillingBenefits/tests/Scenario/SavingsPlansOrderAliasTests.cs
Comment (Yao725) [inline sdk/billingbenefits/Azure.ResourceManager.BillingBenefits/tests/Scenario/SavingsPlansOrderAliasTests.cs:18]:
```suggestion
        private SavingsPlanOrderAliasModelResource _modelResource;
        private TenantResource _tenantResource;
```

File: sdk/billingbenefits/Azure.ResourceManager.BillingBenefits/tests/Scenario/SavingsPlansOrdersTests.cs
Comment (Yao725) [inline sdk/billingbenefits/Azure.ResourceManager.BillingBenefits/tests/Scenario/SavingsPlansOrdersTests.cs:15]:
```suggestion
        private TenantResource _tenant;
```

File: sdk/billingbenefits/Azure.ResourceManager.BillingBenefits/tests/TestHelpers.cs
Comment (Yao725) [inline sdk/billingbenefits/Azure.ResourceManager.BillingBenefits/tests/TestHelpers.cs:24]:
```suggestion
            var request = new SavingsPlanOrderAliasModelData(new BillingBenefitsSku("Compute_Savings_Plan"))
            {
                BillingScopeId = "/subscriptions/eef82110-c91b-4395-9420-fcfcbefc5a47",
                Term = new Term("P3Y"),
                AppliedScopeType = scope,
                DisplayName = "TestSPName" + scope.ToString(),
                BillingPlan = new BillingPlan("P1M"),
                Commitment = new Commitment
                {
                    Grain = "Hourly",
                    CurrencyCode = "USD",
                    Amount = 0.001
                }
            };
```

**Review by Yao725 — COMMENTED** [review]
Please also fix the spell error in CI, you can refer to [this](https://github.com/HarveyLink/azure-sdk-for-net/blob/375265eb56936d43218f902bfb7727027b814129/.vscode/cspell.json) as example.

**Comment by gaoyp830** [comment]
@Yao725 please approve and merge the PR to generate beta SDK if it looks good to you.

**Review by Yao725 — APPROVED** [review]

**Pull request merged**
